mathesar_ui/src/pages/schema/TableCard.svelte

```diff
@@ -206,18 +222,11 @@
   }
 
   /** Menu button =========================================================== */
-  .fake-button {
-    flex: 0 0 auto;
-    width: var(--menu-trigger-size);
-    height: var(--menu-trigger-size);
-  }
-  .hovering-menu-trigger .fake-button {
-    background: var(--slate-100);
-  }
   .menu-container {
     position: absolute;
     top: 0;
     right: 0;
+    margin: 5px;
```

Comment:
Please do not usage absolute units like pixels and prefer relative relative units like rem/em. You can use the existing size variables in the variables.scss file.

Reply:
used `var(--size-ultra-small)` instead